SnipperAPI/Models/models.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace SnipperAPI.Models
+{
+    public class models
```

Comment:
Please remove this double class declaration. Models needs to be a namespace only, and then leaving the class which is RegisterModel. Also, one removed the above declaration, rename the file to RegisterModel.cs :)